src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const promise1 = new Promise((resolve) => {
+  logo.addEventListener('click', () => {
+    resolve();
+  });
+});
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    // eslint-disable-next-line prefer-promise-reject-errors
+    reject();
+  }, 3000);
+});
+
+const handleSuccess = () => {
+  const successMess = document.createElement('div');
+
+  successMess.className = 'message';
+  successMess.textContent = 'Promise was resolved!';
+  document.body.appendChild(successMess);
+};
+const handleError = () => {
+  const errorMess = document.createElement('div');
+
+  errorMess.className = 'message error-message';
+  errorMess.textContent = 'Promise was rejected!';
+  document.body.appendChild(errorMess);
+};
+
+promise1.then(handleSuccess).catch(handleError);
+promise2.then(handleSuccess).catch(handleError);
```

Comment:
Currently, both promises are handled separately. If you want to handle them together, consider using `Promise.all` or `Promise.race` depending on your requirements. This might be useful if you want to perform an action only when both promises are settled.